view/src/App.tsx

```diff
@@ -1,86 +1,148 @@
-import axios from "axios";
-import { useEffect, useState } from "react";
+import React, { useState, useEffect } from "react";
 import Navbar from "./components/Navbar";
-import { API_BASE_URL } from "./consts/consts";
-import GraphPage from "./pages/GraphPage";
-import WelcomePage from "./pages/WelcomePage";
 import { Dataset } from "./types/types";
+import logo from "./logo.svg"; // Replace with your actual image path
 
 function App() {
-  const [page, setPage] = useState<"welcome" | "graph">("welcome");
```

Comment:
The entire main functionality of the app has been deleted here and has not been rewritten